graphdatascience/tests/integration/test_database_ops.py

```diff
@@ -13,6 +13,22 @@
 GRAPH_NAME = "g"
 
 
+@pytest.mark.skip_on_aura
+def test_init_without_neo4j_db(runner: Neo4jQueryRunner) -> None:
+    default_database = runner.database()
+
+    MY_DB_NAME = "bananas"
+    runner.run_cypher("CREATE DATABASE $dbName WAIT", {"dbName": MY_DB_NAME})
+
+    runner.run_cypher("DROP DATABASE $dbName WAIT", {"dbName": default_database})
```

Comment:
smart with integration test. I struggled with the unit test which I attempted first. but it would not have worked anyway